ci/scripts/integration_spark.sh

```diff
@@ -45,6 +45,13 @@ export MAVEN_OPTS="${MAVEN_OPTS} -Dorg.slf4j.simpleLogger.log.org.apache.maven.c
 
 pushd ${spark_dir}
 
+  # Due to CVE-2023-34462 we upgraded to a memory netty version which is incompatible
+  # with previous spark versions. Patch the pom to use newer version.
+  sed -i.bak -E -e \
```

Comment:
I run this `sed` statement locally, but I found that the `netty.version` in the `pom.xml` still `4.1.93.Final `

Maybe we can test `build/mvn  versions:set-property -Dproperty=netty.version  -DnewVersion=4.1.94.Final -DgenerateBackupPoms=false`

Reply:
Hmm, [this line](https://github.com/ursacomputing/crossbow/actions/runs/5532636163/jobs/10095003367#step:6:13303) may also say to use `4.1.93.final`

Reply:
Thanks, I'll investigate, I might have messed up on my tests :)